### PR TITLE
add(epp) Pure Storage monitoring procedure EN

### PR DESCRIPTION
**Pull request by psamecentreon**

### CODE REVIEW

**Comment by Sims24** [comment]
Looks good to me. @thibaults-centreon wdys ?

File: en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+## Prerequisites
+
+### Configuration Pure Storage Equipment  
+
+In order to monitor your Pure Storage equipments the SNMP v2 must be configured.
+
+### Network flow
+
+Your centreon server must be able to reach the PureStorage storage array over UDP/161 SNMP port.
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md:41]:
```suggestion
Your Centreon poller must be able to reach the PureStorage storage array on the UDP/161 SNMP port.
```

File: en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+| pureArrayReadLatency     | Storage array Read Latency (us/op). Unit: us/operations         |
+| pureArrayWriteLatency    | Storage array Write Latency (us/op). Unit: us/operations        |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+### Configuration Pure Storage Equipment  
+
+In order to monitor your Pure Storage equipments the SNMP v2 must be configured.
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md:37]:
```suggestion
The SNMP agent must be configured on the Pure Storage devices.
```

File: en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+<!--Online IMP Licence & IT-100 Editions-->
+
+1. Install the Plugin on every poller expected to monitor Pure Storage SNMP resources:
+
+```bash
+yum install centreon-plugin-Hardware-Storage-Purestorage-Snmp
+```
+
+2. Install the "Pure-Storage-SNMP" Centreon Plugin-Pack from the "Configuration > Plugin packs > Manager" page
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md:55]:
```suggestion
2. Install the "Pure Storage SNMP" Centreon Plugin-Pack from the "Configuration > Plugin packs > Manager" page
```

File: en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+yum install centreon-plugin-Hardware-Storage-Purestorage-Snmp
+```
+
+2. Install the Centreon Plugin Pack RPM on your central server:
+
+```bash
+yum install centreon-pack-hardware-storage-purestorage-snmp
+```
+
+3. Install the "PureStorage SNMP" Centreon Plugin-Pack from the "Configuration > Plugin packs > Manager" page
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md:72]:
```suggestion
3. Install the "Pure Storage SNMP" Centreon Plugin-Pack from the "Configuration > Plugin packs > Manager" page
```

File: en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+<!--Offline IMP License-->
+
+1. Install the Plugin on every poller expected to monitor Pure Storage resources:
+
+```bash
+yum install centreon-plugin-Hardware-Storage-Purestorage-Snmp
+```
+
+2. Install the Centreon Plugin Pack RPM on your central server:
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/hardware-storage-purestorage-snmp.md:66]:
```suggestion
2. Install the Centreon Plugin-Pack RPM on your central server:
```

**Pull request merged**